### ISSUE TITLE
extract_swf.sls Specified sha256 checksum does not match 

### ISSUE THREAD

**Issue by nolsen311** [REMnux/salt-states#208]
I'm attempting to install remnux as an addon to a fresh Ubuntu-20.04 WSL instance.
Have just installed SiFT, restarted and tried to install per docs.
```  
sift install --mode=server  
```  
I have manually downgraded jinja2==3.0.3  
```  
pip install jinja2==3.0.3  
```  
then tried to install remnux as addon
```  
remnux install --mode=addon  
```  

Appears that the stored source_hash does not match the most current version as downloaded from URL.
I think I set up a pull request for this.

error messages from saltstack.log:
```  
    name: /usr/lib/x86_64-linux-gnu/libwebkitgtk-3.0.so.0
    result: true
    start_time: '15:02:56.698708'
  file_|-remnux-addon-version-file_|-/etc/remnux-version_|-managed:
    __run_num__: 759
    __sls__: remnux.addon
    changes: {}
    comment: 'One or more requisite failed: remnux.scripts.remnux-scripts'
    duration: 0.008
    result: false
    start_time: '15:08:49.843576'
--
  file_|-remnux-scripts-extract_swf-source_|-/usr/local/bin/extract_swf.py_|-managed:
    __id__: remnux-scripts-extract_swf-source
    __run_num__: 451
    __sls__: remnux.scripts.extract_swf
    changes: {}
    comment: Specified sha256 checksum for /usr/local/bin/extract_swf.py (665358926164eb2eccfa246ec4ea0b6b59409d90f0086f7e12559586d2195ac9)
      does not match actual checksum (474b4e296be86c245fec8e28e435398c5d6ada736de1552ff21950419ccb282b)
    duration: 238.644
    name: /usr/local/bin/extract_swf.py
    result: false
    start_time: '15:07:50.891844'
--
    name: remnux-rubygems
    result: true
    start_time: '15:07:43.581501'
  test_|-remnux-scripts_|-remnux-scripts_|-nop:
    __run_num__: 574
    __sls__: remnux.scripts
    changes: {}
    comment: 'One or more requisite failed: remnux.scripts.extract_swf.remnux-scripts-extract_swf-source'
    duration: 0.006
    result: false
    start_time: '15:08:23.046288'  
```

**Comment by digitalsleuth** [comment]
Hi @nolsen311 we are tracking the issue from your Pull Request earlier today, we'll be getting on it soon.

**Comment by lennyzeltser** [comment]
Fixed in release [v2022.15.4](https://github.com/REMnux/salt-states/releases/tag/v2022.15.4). Thanks for the report!
